playwright-e2e/tests/smoke-test.spec.ts

```diff
@@ -128,5 +133,10 @@ test("Smoke Test @smoke-test @accessibility", async ({
     contentType: 'application/json'
   });
 
+    const accessibilityScanResults = await makeAxeBuilder()
+  // Automatically uses the shared AxeBuilder configuration,
+  // but supports additional test-specific configuration too
+  .analyze();
+    
```

Comment:
seems like this is different to master:

const accessibilityScanResults = await makeAxeBuilder()
  // Automatically uses the shared AxeBuilder configuration,
  // but supports additional test-specific configuration too
  .analyze();

  await testInfo.attach('accessibility-scan-results', {
    body: JSON.stringify(accessibilityScanResults, null, 2),
    contentType: 'application/json'
  });

Reply:
Removed unwanted code. hope thats ok?